Nos/Service/Analytics.swift

```diff
@@ -302,3 +294,31 @@ class Analytics {
         track("Mentions Autocomplete Opened")
     }
 }
+
+extension Analytics {
+    /// Tracks the event with the given properties in the analytics provider.
+    /// Also calls `trackBreadcrumb` to track this event as a breadcrumb in our error reporting tool (Sentry).
+    /// - Parameters:
+    ///   - eventName: The event name to track.
+    ///   - properties: The properties to include with the event.
+    private func track(_ eventName: String, properties: [String: Any] = [:]) {
+        if properties.isEmpty {
+            Log.info("Analytics: \(eventName)")
+        } else {
+            Log.info("Analytics: \(eventName): \(properties)")
+        }
+        postHog?.capture(eventName, properties: properties)
+
+        trackBreadcrumb(eventName)
+    }
+    
+    /// Adds a breadcrumb for the given event name for tracking in our error reporting tool (Sentry).
+    /// - Parameter eventName: The event for which to add a breadcrumb.
+    private func trackBreadcrumb(_ eventName: String) {
+        let crumb = Breadcrumb()
+        crumb.level = SentryLevel.info
+        crumb.category = "analytics"
+        crumb.message = eventName
+        SentrySDK.addBreadcrumb(crumb)
+    }
```

Comment:
This should be in CrashReporting, and perhaps Analytics can call it. Still not sure if Analytics should depend on CrashReporting but I _am_ sure this code goes in CrashReporting.